### ISSUE TITLE
VLC playing in UE4.13.2 but not visible on texture/material

### ISSUE THREAD

**Issue by RayIon** [ue4plugins/VlcMedia#33]
Hello,

For historic reasons we're using UE4.13.2, so I downloaded your plugin for that tag and put it into our project. We've "forced" VLC3.0.0 onto Ubuntu 14.04.3 and when we run our test level, Vlc reports itself as having opened the videos and playing them, but the media texture/material stays blank.

Any ideas on how to troubleshoot this? It feels like there's some tiny part missing that'd make it all work (for us, I mean).

In short:
 - LibVlc, LibVlcCore and Player all found and instantiated by Unreal
 - Plugins _seem_ to be loading fine, apart from LibVnc and LibJpeg_Plugin -- but they look unrelated?
 - Standalone Vlc plays all the videos fine on that machine
 - No Vlc errors, no Unreal errors, Vlc reports itself as playing and looping the video.

What else can I do? Any suggestions?

**Comment by gmpreussner** [comment]
Oh boy, 4.13 is pretty old at this point. I'm also pretty slammed at the moment, so I can't look into this. You'll have to debug this yourself.

The first thing to do is check whether you're getting any [video callbacks](https://github.com/ue4plugins/VlcMedia/blob/4.13.2/Source/VlcMedia/Private/Player/VlcMediaOutput.cpp#L423) from VLC. The next thing to check is whether the [frames are forwarded](https://github.com/ue4plugins/VlcMedia/blob/4.13.2/Source/VlcMedia/Private/Player/VlcMediaOutput.cpp#L416) to the video sink. If there's no video sink, it means that your MediaPlayer isn't set up correctly. Make sure it has an output texture assigned.

**Comment by RayIon** [comment]
Yeah we're getting callbacks, I've added logging and can see we get "Playing" and related callbacks.

Cheers for the information about forwarded frames, I'll check that out.
